azext_iot/dps/_help.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+"""
+Help definitions for IoT Hub commands.
+"""
+
+from knack.help_files import helps
+
+
+def load_deviceprovisioningservice_help():
+    helps["iot device registration"] = """
+        type: group
+        short-summary: Manage IoT Device registrations through the IoT Device Provisioning Service.
```

Comment:
We need to clearly delineate the operations available in `az iot device registration` and `az iot dps enrollment registration` or `az iot dps enrollment-group registration`. Both this and the other command groups have the same verbiage of managing device registrations

![image](https://user-images.githubusercontent.com/28658112/172499848-9459fe86-3636-4496-ac1b-0b42c4ec0e97.png)

We should describe and elevate the device side nature of this command group.